Bricks/file/file.h

```diff
@@ -54,12 +54,17 @@ namespace current {
 // Platform-indepenent, injection-friendly filesystem wrapper.
 struct FileSystem {
 #ifndef CURRENT_WINDOWS
-  static constexpr char PathSeparatingSlash = '/';
+  static constexpr char PathSeparator = '/';
```

Comment:
Are you suggesting we rename a simple and readable `char` variable the name of which contains `"Slash"` into something more obscure because some _broader communities_ like to do it this way?